test/jit/test_class_type.py

```diff
@@ -1167,13 +1167,19 @@ def free_function(x: int) -> int:
 
         @torch.jit.script
         class Properties(object):
+            __ignored_properties__ = ["unsupported"]
```

Comment:
OOC, is `__ignored_properties__` the API we want users to use?

Reply:
I think so. As @ppwwyyxx mentioned it should be easy to enable ignoring properties with a decorator. Should we do it for this release or wait for the next?

Reply:
I think it should be good to do for this release - i don't think it's particularly risky (although mabye `unused` is a better fit)